lib/mongoid/history/tracker.rb

```diff
@@ -18,8 +18,6 @@ module Tracker
 
         index(scope: 1)
         index(association_chain: 1)
-
-        Mongoid::History.tracker_class_name = name.tableize.singularize.to_sym
```

Comment:
Is anything changing for existing users because of this?

Reply:
I'm guessing this line was originally there so that you don't have to configure a default tracker. It would dynamically set the loaded tracker as default. When multiple trackers got supported, this line no longer works as expected.

So i'm guessing the ones that will be affected are the ones that don't have a default tracker configured and do not have a tracker class set in the trackable options.